### ISSUE TITLE
Loss curve

### ISSUE THREAD

**Issue by acharkq** [facebookresearch/vicreg#4]
Hi, thanks for the great work!

I am trying to implement your method on my own dataset. Would you please post a figure of the loss changes with respect to the training epochs? It would be very helpful. 

Thanks!

**Comment by Adrien987k** [comment]
Hi,
I send a log file to you by e-mail.  If you didn't receive the email contact me at abardes@fb.com.

**Comment by acharkq** [comment]
Thank you! I have received the Email.

**Comment by sanagno** [comment]
Hey! If you can send me the log files for the unsupervised training and the classifier head evaluation, I would be really grateful as well, thanks!

**Comment by alomrani** [comment]
Hello, if possible, please send a log to me as well of SSL pre-training. Thanks

**Comment by deepconsc** [comment]
Hey @Adrien987k, great work! 
I've just shot an email to you asking for the log file. Thanks in advance!

**Comment by JigSaw0331** [comment]
Hello, would you send a log to me as well of SSL pre-training? Thanks!

**Comment by Irislucent** [comment]
Dear author, would you please send me the log file to help with my tuning? Thank you!

**Comment by gurdeepmaurya** [comment]
Hey @acharkq Could you forward me the pre-training log file at my email id [gurdeepmaurya@gmail.com]
